app/modules/team-member/service.server.ts

```diff
@@ -161,6 +163,14 @@ export async function getTeamMembers(params: {
         orderBy: { createdAt: "desc" },
         include: {
           custodies: true,
+          user: {
```

Comment:
Shouldnt we make this an optional param of `getTeamMembers`. I think there are places where we use it where we dont want the users. And this is a join so it can be slowing down.